app/javascript/mastodon/features/terms_of_service/index.jsx

```diff
@@ -46,12 +46,11 @@ class TermsOfService extends React.PureComponent {
               represent that you are authorized to and intend to bind that
               entity.</p>
 
-            <h2 id='fxa-account'>You'll Need A Firefox Account</h2>
+            <h2 id='mozilla-account'>You'll Need A Mozilla Account</h2>
```

Comment:
Updated the ID of the element to drop FxA from the name, too. Double-checked that nothing in Mastodon or Elk code actually references it yet. I expect this header ID is more for future-proofing - to be able to link straight to a section of the terms, e.g. `terms#mozilla-account`.